test/functional/p2p_blocksonly.py

```diff
@@ -3,7 +3,6 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test p2p blocksonly"""
-
```

Comment:
Travis seems to be passing, but there is a linter error. Potentially pep8. Maybe it is this missing line?

You can try to format the python code with some kind of formatter or run the `./test/lint/lint-python.sh` python and fix the issues up manually.